kcbq-connector/test/resources/connector-template.properties

```diff
@@ -26,9 +26,12 @@ maxWriteSize=1000
 tableWriteWait=1000
 
 keyfile=/tmp/key.json
-schemaRetriever=com.wepay.kafka.connect.bigquery.schemaregistry.schemaretriever.SchemaRegistrySchemaRetriever
-schemaRegistryLocation=http://schema-registry:8081
+schemaRetriever=com.wepay.kafka.connect.bigquery.retrieve.IdentitySchemaRetriever
 
 enableBatchLoad=kcbq_test_gcs-load
 batchLoadIntervalSec=10
 
+transforms=RegexTransformation
+transforms.RegexTransformation.type=org.apache.kafka.connect.transforms.RegexRouter
+transforms.RegexTransformation.regex=.*
+transforms.RegexTransformation.replacement=$0
```

Comment:
What's this supposed to do? AFAICT it's going to grab the entire topic (matched by `.*`), and then replace it with the entire match (`$0`)... so will this have any effect on the records here?

Reply:
No, this will not have any effect on the records. The regex was just meant to be an example/placeholder :)

Reply:
Ah, gotcha. In that case, can we do one of the following (no string preference on my part for/against any of them):
- Add a comment explaining what this is and how someone might alter it to achieve table routing behavior
- Remove it
- Alter it to provide a more practical example (such as appending or stripping a prefix or suffix, or redirecting all records from one topic to a different topic but leaving all others unaffected)

Reply:
I vote for:

> Alter it to provide a more practical example (such as appending or stripping a prefix or suffix, or redirecting all records from one topic to a different topic but leaving all others unaffected)